src/pages/index.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+
+// import useHomeStyles from './index.styles'
+import { Layout } from '../components/layout/layout.component'
+import Title from '../components/shared/title/title.component'
+import Button from '../components/shared/button/button.component'
+import Row from '../components/shared/row/row.component'
+import Col from '../components/shared/col/col.component'
+import Section21 from '../assets/images/home/section2_1.inline.svg'
+import Section22 from '../assets/images/home/section2_2.inline.svg'
+import Section23 from '../assets/images/home/section2_3.inline.svg'
+// import Section3 from '../assets/images/home/section3.inline.svg'
+
+const Home = () => {
+  // const classes = useHomeStyles()
```

Comment:
I would remove this styles from here and keep a file for each route in the `pages` folder. Styles should be applied on the components used in a page.

Reply:
New component Home created. Check if that was the idea you had.